docs/guides/key-manager/execute-relay-call.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_label: 'Execute Relay Call'
+sidebar_position: 2
+---
+
+# Execute Relay Call
+
+The [LSP6 KeyManager standard](../../standards/universal-profile/lsp6-key-manager.md) enables anybody to execute a transation on behalf of a Universal Profile, given they have a valid transaction which has been signed by a key which controls the Universal Profile.
```

Comment:
Is there already a set convention for LSP terms @CJ42 ? Otherwise I would suggest to either write `LSP6 KeyManager Standard` , or keep the word `standard` lowercase but out of the link.